terraform/root/aws-dm-dev/main.tf

```diff
@@ -8,3 +8,9 @@ module "aws_env" {
   aws_main_account_id = "${var.aws_main_account_id}"
   aws_dev_account_id = "${var.aws_dev_account_id}"
 }
+
+module "s3-bucket" {
```

Comment:
If I rename the module to something less broad, like `dev-s3-bucket`, does it make sense to just define the bucket resource within `root/aws-dm-dev` as it's the only environment where it's actually going to be used? Or is it better practise to define everything in modules and import them, even if that module would only ever be used once?